src/Data/Auth/Providers/DatabaseAuthProvider.tsx

```diff
@@ -10,16 +11,23 @@ import { AuthContext } from "../AuthContext";
 export const DatabaseAuthProvider: React.FC<React.PropsWithChildren> = ({ children }) => {
   const [user, setUser] = useState<string | null>(null);
   const navigate = useNavigate();
+  const baseUrlManager = new PrimaryBaseUrlManager(
```

Comment:
It's optional as I don't know how much work that would add but maybe we could ad `navigateTo("Login", undefined);` there, I don't remember if the path is there but that would align with what we have in other places.

Reply:
Was my first thought too, and I tried it, but to work properly, the login page needs to be outside the RouteOutlet. It also means we can't add it to the PageDictionary/PrimaryPageManager directly. The RouteOutlet makes use of the Initializer, which expects the environement endpoint to be called, resulting the login page to also require that endpoint to return something else than a 401. You end up in a loop. To change this behaviour, would require a lot of work compared to the simpler approach of keeping the Login Route outside the dictionary of pages.

This also responds to the navigateTo useage, which requires the page to be part of the PrimaryPageManager :)